composer.json

```diff
@@ -14,7 +14,7 @@
     "require-dev": {
         "cakephp/bake": "dev-cake5",
```

Comment:
Shouldn't this be updated too?

Reply:
yup, I missed that one 😄 
seems that packagist cached that version string